src/Object/Contact.php

```diff
@@ -720,21 +721,22 @@ public static function getPostsFromUrl($contact_url)
 
 		// There are no posts with "uid = 0" with connector networks
 		// This speeds up the query a lot
-		$r = dba::select('contact', array('network', 'id AS author-id', 'contact-type'), array('nurl' => normalise_link($contact_url), 'uid' => 0));
+		$s = dba::select('contact', array('network', 'id AS author-id', 'contact-type'), array('nurl' => normalise_link($contact_url), 'uid' => 0));
+		$r = dba::inArray($s);
```

Comment:
When you are using ```dba::inArray``` it returns an array structure like the ```q``` function, so you had to use ```$r[0]``` later in the code. But since this call will only use the first queried row, it will be better to use ```array('limit' => 1)``` for the ```dba::select``` and use ```$r``` like you have already done in the code.